client/src/components/Profile/ProfileContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Profile from './Profile';
+import { getUser } from '../../store/profile/actions';
+
+class ProfileContainer extends React.Component<any> {
```

Comment:
any?
it will be IProps = IStateToProps & IDisaptchToProps or something like this

Reply:
Done